xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -1587,3 +1586,32 @@ void CXBMCApp::surfaceDestroyed(CJNISurfaceHolder holder)
 
   m_window.reset();
 }
+
+bool CXBMCApp::isHeadsetPlugged()
```

Comment:
namespace
{
bool IsHeadsetPlugged()
{
...
}
} // namespace

Reply:
ok, move to an unnamed namespace